Content.Server/Radio/Components/RadioMicrophoneComponent.cs

```diff
@@ -34,6 +34,12 @@ public sealed partial class RadioMicrophoneComponent : Component
     [DataField("toggleOnInteract")]
     public bool ToggleOnInteract = true;
 
+    /// <summary>
+    // The radio frequency on which the message will be transmitted
+    /// </summary>
+    [DataField("frequency")]
```

Comment:
```suggestion
    [DataField]
```
modern format no longer requires a title

you can also remove it from the fields above.